src/sagas/networkSettings.js

```diff
@@ -148,15 +144,15 @@ export function* updateNetworkSettings(action) {
     txMiningServiceUrl: networkSettings.txMiningServiceUrl,
   };
 
+  config.setTxMiningUrl(txMiningServiceUrl);
   config.setExplorerServiceBaseUrl(explorerServiceUrl);
   config.setServerUrl(nodeUrl);
-  config.setTxMiningUrl(txMiningServiceUrl);
 
   // - walletServiceUrl has precedence
   // - nodeUrl as fallback
   let potentialNetwork;
   let network;
-  if (walletServiceUrl && useWalletService) {
+  if (useWalletService && !isEmpty(walletServiceUrl)) {
```

Comment:
Oh, we were setting the url to null?

Ok, so this will now just fallback to the URL in the constants file

Reply:
No, if `walletServiceUrl` is empty we will set an empty value in redux state. When the wallet reloads and wallet service is enabled, it will try to connect but will fail, then it will fallback to fullnode.